TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/http/NettyJsonContentAuthSigner.java

```diff
@@ -12,35 +12,49 @@
 public class NettyJsonContentAuthSigner extends ChannelInboundHandlerAdapter {
     IAuthTransformer.StreamingFullMessageTransformer signer;
     HttpJsonMessageWithFaultingPayload httpMessage;
-    List<HttpContent> receivedHttpContents;
+    List<HttpContent> httpContentsBuffer;
```

Comment:
why was this renamed to 'Buffer' if it still stores HttpContents?

Reply:
This is what helped me understand what is happening in this handler.

HttpContents are read into this handler and buffered from flowing to the other handlers until they are all in and the headers are signed, then we send the updated headers and all the contents.

Reply:
How about something like deferredHttpContents or receivedHttpContentsToSendAfterComputation?  Yeah, those are long, but they could really clear up confusion.  Buffer has 2 alternate meanings here.  One is for a byte buffer and the other is something that you're holding only to batch some operations for efficiency.